service/grpc/topic.go

```diff
@@ -62,13 +63,28 @@ func (s *Server) OnTopicEvent(ctx context.Context, in *pb.TopicEventRequest) (*p
 	}
 	key := fmt.Sprintf("%s-%s", in.PubsubName, in.Topic)
 	if h, ok := s.topicSubscriptions[key]; ok {
+		data := interface{}(in.Data)
+		if len(in.Data) > 0 {
+			switch in.DataContentType {
+			case "application/json":
```

Comment:
It can also be "application/json; UTF=8", so you probably need to normalize the input.

Reply:
I updated this code to use `mime.ParseMediaType(in.DataContentType)` for this. For now it only handles `application/json`. Detecting different prefixes/suffixes for JSON can come in future PRs if it makes sense.